docs/reference/edot-collector/components/elasticsearchexporter.md

```diff
@@ -135,11 +135,43 @@ The exporter supports both internal batching and OpenTelemetry's standard `sendi
 
 ### Internal batching (default)
 
-By default, the exporter performs its own buffering and batching, as configured through the `flush` setting, unless the `sending_queue` and  `batcher` settings are defined.
+By default, the exporter performs its own buffering and batching, as configured through the `flush` setting, unless the `sending_queue` and/or  `batcher` settings are defined.
 
 ### Using sending queue
 
-The Elasticsearch exporter supports the `sending_queue` setting, which supports both queueing and batching. However, the sending queue is currently deactivated by default. You can turn on the sending queue by setting `sending_queue` to true. Batching support in sending queue is also deactivated by default and can be turned on by defining `sending_queue::batch`. For example:
+```{applies_to}
+stack: 9.0 9.1
+```
+
+The sending queue can be enabled and configured with the `batcher` section, using [common `batcher` settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/internal/queue_sender.go).
```

Comment:
This is not technically correct. In previous versions, `sending_queue` and `batcher` are both supported, but they are separate configs configuring the queue and batch correspondingly.

Reply:
thanks I only have one doubt: `sending_queue` does not seem to be aviable before v0.132.0.
if that's correct, we should not include a sending_queue section for 9.0 and 9.1.

I was thinking to structure the document with separate sections about sending queue and batching, but only if sending queue is available in 9.0 and 9.1

Reply:
nvm I just re-read the docs in the upstream repo, `sending_queue` was supported in older versions too.
I'll update with dedicated sections